### PR TITLE
Double hit

### PR DESCRIPTION
**Pull request by Artemis-chan**
Implements doubles(#5)
The placeholder art(i know its crap, but its placeholder) should be addressed along with #7

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
             "request": "launch",
             "preLaunchTask": "build",
             // If you have changed target frameworks, make sure to update the program path.
-            "program": "${workspaceFolder}/osu.Game.Rulesets.Diva.Tests/bin/Debug/netcoreapp3.1/osu.Game.Rulesets.Diva.Tests.dll",
+            "program": "${workspaceFolder}/osu.Game.Rulesets.Diva.Tests/bin/Debug/net5.0/osu.Game.Rulesets.Diva.Tests.dll",
```
Comment (Thesola10) [inline .vscode/launch.json:13]: Do we really need to push `.vscode`? It's not important for the project

File: osu.Game.Rulesets.Diva/Objects/Drawables/DrawableDivaHitObject.cs [hunk 2]
```diff
@@ -31,16 +31,16 @@ public class DrawableDivaHitObject : DrawableHitObject<DivaHitObject>, IKeyBindi
...
-        private readonly DivaAction validAction;
+        protected readonly Sprite approachHand;
+        protected readonly ApproachPiece approachPiece;
+
+		protected readonly DivaAction validAction;
 
-        private bool validPress = false;
-        private bool pressed = false;
+        protected bool validPress = false;
+		protected bool pressed = false;
```
Comment (Thesola10) [inline osu.Game.Rulesets.Diva/Objects/Drawables/DrawableDivaHitObject.cs:40]: careful with indents ;)

**Review by Thesola10 — APPROVED** [review]
Looking good, works without issues

File: .vscode/launch.json (hunk 1 above)
Comment (Artemis-chan) [reply .vscode/launch.json:13]: its not necessary but still useful

**Pull request merged**